velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -643,6 +672,13 @@ bool ExpressionFuzzer::isDone(size_t i, T startTime) const {
   return i >= FLAGS_steps;
 }
 
+void ExpressionFuzzer::reset() {
+  VELOX_CHECK(inputRowTypes_.empty());
+  VELOX_CHECK(inputRowNames_.empty());
```

Comment:
nit: I would suggest to move these checks outside of reset() because they don't seem to reset anything.

Reply:
they dont reset anything but including them here signifies state that should be empty if reset is called. Currently these are std::moved when an input row is generated which is why they are empty but having them here ensures this will be true if reset is called, so in the future if something changes that does not reset these before this, then we can catch it and make sure we reset it here.